### PR TITLE
Optimize code.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -1365,12 +1365,15 @@ protected function enumOptions(string $enumClass): array
     {
         assert(is_subclass_of($enumClass, BackedEnum::class));
 
+        $hasLabel = is_a($enumClass, EnumLabelInterface::class, true) || method_exists($enumClass, 'label');
```
Comment (othercorey) [inline src/View/Helper/FormHelper.php:1368]: Do we really need both checks? We can just check for label method for both, right?

Comment (dereuromark) [reply src/View/Helper/FormHelper.php:1368]: The first is probably faster^^

Comment (ADmad) [reply src/View/Helper/FormHelper.php:1368]: IIRC the double checks are cause the interface was added at bit later after the 5.0 release, I guess we can drop the method check in a new minor.

Comment (markstory) [reply src/View/Helper/FormHelper.php:1368]: I agree that one check would be faster than two, but I don't think we need to over-correct on efficiency here. Compatibility throughout the major is more important.

**Pull request merged**